src/app_service/service/community/service.nim

```diff
@@ -1035,7 +1031,8 @@ QtObject:
         int(cropRectJson["y"].getFloat() + cropRectJson["height"].getFloat()),
         $bannerJson,
         historyArchiveSupportEnabled,
-        pinMessageAllMembersEnabled)
+        pinMessageAllMembersEnabled,
+        self.communities[id].encrypted)
```

Comment:
Why do we need to pass the encrypted param to edit? I thought the goal was that we can't edit that?

Reply:
removed